clients/java/dkv-client/src/test/resources/dkv_config.yaml

```diff
@@ -23,3 +22,12 @@ nexus-repl-timeout : 5
 nexus-lease-based-reads : true
 nexus-snapshot-count : 100
 nexus-snapshot-catchup-entries : 5
+discovery-service:
+  server-config:
+    status-ttl: 300
+    heartbeat-timeout: 60
+  client-config:
+    discovery-service-addr: "multi:///127.0.0.1:8001,127.0.0.1:8002,127.0.0.1:8003"
+    push-status-interval: 1
+    poll-cluster-info-interval: 1
```

Comment:
This needs also to be done in dkvsrv.yaml at top level.

Reply:
yes had already done that